.github/workflows/dask_runner_tests.yml

```diff
@@ -22,7 +22,7 @@ name: Dask Runner Tests
 on:
   schedule:
     - cron: '3 7 * * *'
-  pull_request:
+  pull_request_target:
```

Comment:
change to "pull_request_target" will make the workflow runs on HEAD instead of the PR branch. Is there any issue using "pull_request" trigger?

Reply:
No issue at all, happy to change it.